File: library/hlscache/src/main/java/com/kingz/mobile/libhlscache/bean/CacheVideoConfig.java
package com.kingz.mobile.libhlscache.bean;

/**
 * 影片配置。
 * Created 2017/11/8.
 */
public class CacheVideoConfig {
    private int forwardCacheTime;  // 单位秒，上取整

    public CacheVideoConfig(int forwardCacheTime) {
        this.forwardCacheTime = forwardCacheTime;
    }

    public CacheVideoConfig() {
        forwardCacheTime = Integer.MAX_VALUE;
    }

    public int getForwardCacheTime() {
        return forwardCacheTime;
    }
}

File: library/hlscache/src/main/java/com/kingz/mobile/libhlscache/bean/CacheVideoConfig.kt
package com.kingz.mobile.libhlscache.bean

/**
 * 缓存影片配置
 */
class CacheVideoConfig {
    var forwardCacheTime: Int // 单位秒，上取整
        private set

    constructor(cacheTime: Int) {
        forwardCacheTime = cacheTime
    }

    constructor() {
        forwardCacheTime = Int.MAX_VALUE
    }

}